org.adempiere.pipo.handlers/src/org/adempiere/pipo2/handler/ColumnElementHandler.java

```diff
@@ -348,6 +349,14 @@ public void create(PIPOContext ctx, TransformerHandler document)
 		addTypeName(atts, "table");
 		document.startElement("", "", I_AD_Column.Table_Name, atts);
 		createColumnBinding(ctx, document, m_Column);
+		
+		ctx.ctx.put("Table_Name",X_AD_Column.Table_Name);
+		try {
+			new CommonTranslationHandler().packOut(ctx.packOut, document, null, AD_Column_ID);
+		} catch(Exception e) {
+			if (log.isLoggable(Level.INFO)) log.log(Level.INFO, e.getMessage());

```

Comment:
Should be log.log(Level.INFO, e.getMessage(), e). As I've mention before, StackTrace matter. Same goes for the log below